redfish/virtual_media.go

```diff
@@ -0,0 +1,81 @@
+//
```

Comment:
Please rename file to virtualmedia.go to match the object name.

Reply:
If you run tools/gen_schema.sh, that will generate this file under gofiles and will automatically pull in the spec descriptions as docs for each public struct and field.